### ISSUE TITLE
Heterogeneous Narwhal TeX Spec

### ISSUE THREAD

**Issue by isheff** [anoma/research#9]
Get the [Heterogeneous Narwhal TeX Spec](https://github.com/anoma/research/tree/master/distributed-systems/heterogeneous-narwhal) in a state where at least other Anoma researchers can tell how it is supposed to work.

**Comment by cwgoes** [comment]
@heindel let's move this issue

**Comment by heindel** [comment]
> @heindel let's move this issue
> 
> also, maybe we can publish Heterogeneous Narwhal initially as an ART report?


yes, in principle, this should be not too much work; moreover, some of the diagrams could be re-used verbatim (or maybe even should?)

@cwgoes  where should we move this issue to (also cc @isheff )?
